python/tutorials/06-fused-attention.py

```diff
@@ -314,31 +490,63 @@ def test_op(Z, H, N_CTX, D_HEAD, P_SEQ, causal, dtype=torch.float16):
     q = torch.empty((Z, H, N_CTX, D_HEAD), dtype=dtype, device="cuda").normal_(mean=0., std=0.5).requires_grad_()
```

Comment:
Can we rename `test_op` to `test_op_fwd`?

Reply:
Sure, will change. I wasn't sure if it'd break any existing scripts that used this test name...